packages/iTwinUI-react/src/core/Table/columns/actionColumn.tsx

```diff
@@ -36,6 +36,8 @@ const ACTION_CELL_ID = 'iui-table-action';
  */
 export const ActionColumn = <T extends Record<string, unknown>>({
   columnManager = false,
+  className = '',
+  style = {},
 } = {}) => {
   return {
     id: ACTION_CELL_ID,
```

Comment:
To confirm, is this a correct way to add new parameters? Because I was expecting to do something like

```tsx
export const ActionColumn = <T extends Record<string, unknown>>({
  columnManager: boolean,
  className: string,
  style: object,
} = {}) => {
```

i.e. I thought we have to pass the variable types in tsx, but they were giving this error

![image](https://user-images.githubusercontent.com/45748283/182904176-f9ad1ba0-4a3d-4515-a5d4-f01690310474.png)